tools/rna_tools/rnaz/rnazAnnotate.xml

```diff
@@ -0,0 +1,50 @@
+<tool id="rnazAnnotate" name="AnnotateRNAz" version="0.1.0">
+    <requirements>
+        <requirement type="package">rnaz</requirement>
```

Comment:
this needs a version number

Reply:
there is just one rnaz version available, still needed?

Reply:
well, you are an optimist ;)